src/kibana/components/vislib/visualizations/tile_map.js

```diff
@@ -47,29 +42,38 @@ define(function (require) {
      * @return {Function} Creates the map
      */
     TileMap.prototype.draw = function () {
+
       var self = this;
       var $elem = $(this.chartEl);
       var div;
-      var worldBounds = L.latLngBounds([-90, -180], [90, 180]);
-
+      var worldBounds = L.latLngBounds([-90, -220], [90, 220]);
 
       // clean up old maps
-      _.invoke(self.maps, 'destroy');
+      _.invoke(self.maps, self.destroy);
```

Comment:
This will iterate over every map and call `destroy` - which itself iterates over every map to destroy it. You don't need `invoke` or iteration here - `self.destroy()` should be enough.